src/ext/framework/help_commands.rs

```diff
@@ -21,18 +34,34 @@ pub fn with_embeds(ctx: &Context,
         let name = args.join(" ");
 
         for (group_name, group) in groups {
-            let mut found: Option<(&String, &Command)> = None;
+            let mut found: Option<(&String, &InternalCommand)> = None;
 
             if let Some(ref prefix) = group.prefix {
                 for (command_name, command) in &group.commands {
-                    if name == format!("{} {}", prefix, command_name) {
-                        found = Some((command_name, command));
+                    if name == format!("{} {}", prefix, command_name) || name == *command_name {
+                        match *command {
+                            CommandOrAlias::Command(ref cmd) => {
+                                found = Some((command_name, cmd));
+                            },
+                            CommandOrAlias::Alias(ref name) => {
+                                error_embed(ctx, message, &format!("Did you mean {:?}?", name));
```

Comment:
Are you meaning to use the debug formatter here? I'd rather not really on implicit behaviour for production (quotation marks on string debug)